pkgs/applications/graphics/grafx2/default.nix

```
@@ -1,29 +1,23 @@
-{ lib, stdenv, fetchurl, SDL, SDL_image, SDL_ttf, zlib, libpng, pkg-config, lua5 }:
+{ lib, stdenv, fetchurl, SDL, SDL_image, SDL_ttf, zlib
+, libpng, libtiff, pkg-config, lua5, fontconfig
+}:
```

Since we are here already, what about one element per line?

```suggestion
{ lib
, stdenv
, fetchurl
, SDL
, SDL_image
, SDL_ttf
, fontconfig
, libpng
, libtiff
, lua5
, pkg-config
, zlib
}:
```

---

Done.